src/components/charts/other.js

```diff
@@ -262,11 +273,34 @@ export default class Other extends Component {
   }
 
   updateBoxplotConfiguration() {
+    const { options, configurations } = this.state;
+
+    options.series = generateBoxplotSeries(this.boxplotData, configurations.boxplot);
+    options.yAxis.plotLines = configurations.boxplot.showAverage ? averageLineBoxplot(this.boxplotData) : [];
+
     this.setState({ rerenderChart: true }, () => {
```

Comment:
I need to add here options also, sot setState will change it properly instead of just mutating the state directly...